distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -343,6 +343,40 @@ java_instance_check value typ =
 is_an : Any -> Any -> Boolean
 is_an value typ = is_a value typ
 
+## UNSTABLE
+   ADVANCED
+
+   Returns the type of the given value.
+
+   Arguments:
+   - value: the value to get the type of.
+type_of : Any -> Any
+type_of value =
+    if is_error value then Base.Error else
+        case value of
+            Base.Polyglot -> Meta.type_of_polyglot_builtin value
```

Comment:
If we delete this _case branch_ and make sure `Meta.type_of_builtin` handles polyglot, the behavior shall stay the same, right?

Reply:
Yes, i changed the definition of `type_of_builtin`